registry/consul/service_discovery.go

```diff
@@ -38,6 +38,7 @@ import (
 	"github.com/apache/dubbo-go/common/extension"
 	"github.com/apache/dubbo-go/common/logger"
 	"github.com/apache/dubbo-go/config"
+	_ "github.com/apache/dubbo-go/metadata/mapping/memory"
```

Comment:
Why import `_ "github.com/apache/dubbo-go/metadata/mapping/memory"`?

Reply:
It's needed when testing consul service discovery. If not, It will panic.